### ISSUE TITLE
Map events' unfairness can be alleviated

### ISSUE THREAD

**Issue by Someone65** [Sumwunn/DoomRPG#51]
I got into a situation where I got put into a toxic map event on map03 of Doom 2. The thing is that it was a level 3 version of the event and I had no radiation suit. I spent 20 whole minutes trying and trying to get a battery quickly before I died and nothing I did would work. Going back to the base wasn't an option because I got silenced in the previous map for an extremely long period of time.

I think map events should have some leeway in the sense that you will be given a grace period before everything goes to shit. For example, the generator of my mentioned event could have some batteries to start with so you're given some time to find batteries before it immediately starts hurting you. The even that turns the map dark could maybe start out with light for a bit before ominously having the lights turn off, etc.

**Comment by Demiosis** [comment]
Nah, next time just buy a detox stim pop it (to remove ALL debuffs including silence) and go back to base.. XD
